public/components/integrations/components/setup_integration_inputs.tsx

```diff
@@ -95,7 +98,9 @@ const suggestDataSources = async (
         }) ?? []
       );
     } else if (type === 's3' || type === 'securityLake') {
-      const result = (await http.get(DATACONNECTIONS_BASE)) as Array<{
+      const result = (await http.get(
+        `${DATACONNECTIONS_BASE}/dataSourceMDSId=${dataSourceMDSId ?? ''}`
```

Comment:
Is this usage correct? This is a URL with an `=` in it instead of a request param? The `http.get` method already has a way of using request params with automatic undefined-handling.

Reply:
wanted to keep it consistent to how other plugins were using and by adding the mds-id here the path is getting differentiated for the other locations where this api is being used